ReleaseProcessAutomation/Scripting/MSBuildCallAndCommit.cs

```diff
@@ -56,8 +57,15 @@ public int CallMSBuildStepsAndCommit (MSBuildMode msBuildMode, SemanticVersion v
 
     if (string.IsNullOrEmpty(msBuildPath))
     {
-      _log.Warning("No MSBuild Path specified in config, will continue without MSBuild");
-      _console.WriteLine("There was no MSBuildPath specified in the config, will continue without Invoking MSBuild");
+      _log.Warning("No MSBuild Path specified in config, will continue without MSBuild.");
+      _console.WriteLine("There was no MSBuildPath specified in the config.\nWill continue without invoking MSBuild.");
+      return -1;
+    }
+
+    if (!File.Exists(msBuildPath))
+    {
+      _log.Warning("The configured MSBuildpath '{MSBuildpath}' does not exist.", msBuildPath);
```

Comment:
maybe it's a syntax-highlight thingy, but there's `{MSBuildPath}` and `msBuildPath` in the code. shouldn't those be the same variable/field/property?

Reply:
Not sure if this works, as I think I tested it like that already. Needed more info the in the message anyways, therefore changed to enumeration